changelogs/current.yaml

```
@@ -267,6 +267,12 @@ bug_fixes:
 - area: lua
   change: |
     Fixed a bug where the user data will reference a dangling pointer to the Lua state and cause a crash.
+- area: ext_authz
+  change: |
+    Fixed fail-open behaviour of the failure_mode_allow config option
+    https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/http/ext_authz/v3/ext_authz.proto
```

Sorry I wasn't clear about the link. It should be something like 
```suggestion
    Fixed fail-open behaviour of the :ref:`failure_mode_allow config option 
    <envoy_v3_api_field_extensions.filters.http.ext_authz.v3.ExtAuthz.failure_mode_allow>`
```